README.md

```diff
@@ -135,6 +135,16 @@ It is possible to list all of the versions of `slycot` available on your platfor
 conda search slycot --channel conda-forge
 ```
 
+Changing the BLAS implementation
+================================
+
+Slycot is compiled and tested against the reference BLAS implementation in conda-forge. The [conda-forge knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html#switching-blas-implementation) shows how to change BLAS implementations.
+
+If you switched to a different implementation, please verify that Slycot is functional, and run its tests with the following command:
+
+```
+pytest --pyargs slycot
+```
```

Comment:
I think conda smithy always overwrites this. Sorry, my smithy call just did. We need to have this info somewhere else. Not sure of the best place.

Reply:
Maybe the best spot is in the Slycot readme.